src/commands/uninstall-deps.yml

```diff
@@ -1,11 +1,34 @@
-description: "Uninstall packages from requirements.txt via Pip."
+description: >
+  Uninstall global packages via pip with requirements.txt file. Override the requirements.txt with a string of package names.
+
+# we could also delete this, I can't imagine a need to remove a pacakge
```

Comment:
I'm in favour of this. I can't imagine an instance where this would have to happen.